### DISCUSSION TITLE
Automatic redirect on 404

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1863] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Whenever someone tries to open a wrong url, when they land on the 404 page, it should redirect them to a specific link or just the home link after a few seconds.

### Operations you have already tried

I thought of adding a script, but the 404.html looks more like a markdown file so no idea how to do it.

**Comment by kungfux** [comment]
One of the possible ways is to override `404.html` to add a script that will redirect to the desired URL like follows:

```html
<script>
window.location.href = "/";
</script>
```

**Reply by ghost** [reply]
This works but when I use the script 
The default 404 text is not shown in the page.
I can't see any reason why this happens

**Reply by kungfux** [reply, marked as answer]
I assume this happens because you use the newest variable `site.data.locales[lang].not_found.statement` (**statement**) instead of `site.data.locales[lang].not_found.statment` (**statment**) that contained a typo and was [fixed](https://github.com/cotes2020/jekyll-theme-chirpy/commit/e6532ad8647adc4fe9aac729eecab599821fd6e2) but not released yet.

**Comment by ghost** [comment]
I see 
Thanks 😊
